modules/planning/common/st_graph_data.h

```diff
@@ -71,6 +90,9 @@ class StGraphData {
   double path_length_by_conf_ = 0.0;
   double total_time_by_conf_ = 0.0;
   planning_internal::STGraphDebug* st_graph_debug_ = nullptr;
+
+  std::vector<std::tuple<double, double, double>> s_boundary_;
+  std::vector<std::tuple<double, double, double>> v_limits_;
```

Comment:
Could we possibly use a proto message on defining the the tuple included in s_boundary_? so that we can combine the tuples in s_boundary_ and v_limits_ together

Reply:
fixed this.